server/src/testFixtures/java/org/elasticsearch/test/IntegTestCase.java

```diff
@@ -1419,7 +1419,7 @@ public String pgUrl() {
 
                 @Override
                 public Sessions sqlOperations() {
-                    return cluster().getInstance(Sessions.class);
+                    return cluster().getInstance(Sessions.class, nodeName);
```

Comment:
This isn't related to the sys.health issue, I came across this by coincidence. Looks wrong to me.